examples/binaries/read-big-state/src/lib.rs

```diff
@@ -69,18 +67,13 @@ mod wasm {
     static mut STATE: Option<State> = None;
```

Comment:
Is there any point to have as `Option` at all? Why can't it be initialized statically?

Reply:
yeah it may have const constructor but see no difference for test